sys/kern/tty.c

```diff
@@ -862,7 +862,11 @@ static int tty_ioctl(file_t *f, u_long cmd, void *data) {
 
 static int tty_vn_getattr(vnode_t *v, vattr_t *va) {
   memset(va, 0, sizeof(vattr_t));
-  va->va_mode = S_IFCHR;
+  /* XXX assume root owns tty and only root can read and write to it */
+  va->va_mode =
+    S_IFCHR | S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP | S_IROTH | S_IWOTH;
```

Comment:
Doesn't `S_IROTH | S_IWOTH` mean that anyone can read and write to it?

Reply:
Yes. Should I change mode of tty?

Reply:
The comment above says that only root can read and write to the tty, so I suggest changing the comment.

Reply:
Yes. I forgot to change comment. Thanks!